src/utils/client.js

```diff
@@ -1,7 +1,9 @@
+// @flow
+
 const { firefox } = require("devtools-client-adapters");
 const { prefs } = require("./prefs");
 
-async function onFirefoxConnect(actions) {
+async function onFirefoxConnect(actions: Object) {
```

Comment:
I'm not sure if typing these as `Object` really buys us much, could we use a proper type that maps to the actions the client receives @jasonLaster